### ISSUE TITLE
Hangs at start up

### ISSUE THREAD

**Issue by thmxv** [CloudCompare/CloudCompare#431]
I am running Windows 10 64bit with a NVIDIA GTX970 and all versions of CloudCompare hangs at startup. Windows says the application is 'Not Responding' and no error message is printed to the console.
I tried several version from 2.6.0 to 2.8 beta, install-able or portable archive, stereo or not. The only versions that worked where the 2.6.2 32bit (x86).

**Comment by cloudcompare** [comment]
Interesting... And sadly I'm clueless.

Have you updated your graphic card driver recently?

**Comment by thmxv** [comment]
Hi, sorry to respond so late, I did see the email until now. I update my drivers regularly and the issue is present since I began to use CC a few months ago, a time period during which I made several driver updates and it did not change the symptoms.

The 32bit version works fine.
All the 64bit versions hangs. The first time during the splash screen but after killing the app and restarting,  the main window pops up as expected and the mouse and menu are accessible for a very short period (one or 2 second) before hanging.

I have no time to compile and debug the executable to provide more info, maybe in the future.
